### PR TITLE
Update problem.md

### PR DESCRIPTION
**Pull request by austinmcorso**
After running into an issue with Express unable to find Jade, I thought this may be helpful for others.

### CODE REVIEW

File: exercises/jade/problem.md [hunk 1]
```diff
@@ -49,3 +49,5 @@ without the time.
 When creating your projects from scratch, install the `jade` dependency with npm.
 
 Again, the port to use is passed by {appname} to the application as `process.argv[2]`.
+
+If you receive `Error: Cannot find module 'jade'`, it is because Express is looking for Jade relative to its path. You can fix this by running `npm install Jade'.
```
Comment (azat-co) [inline exercises/jade/problem.md]: can you fix `npm install Jade` into `npm install jade`? and fix the closing '. it should be ` not '

Comment (marocchino) [inline exercises/jade/problem.md]: end quote should use ` instead of '.

```diff
-`npm install Jade'.
+`npm install Jade`.
```

**Comment by azat-co** [comment]
@austinmcorso if you fix the aforementioned typo, I'll merge the request. thank you!

**Comment by austinmcorso** [comment]
Updated!

**Comment by azat-co** [comment]
@austinmcorso thank you!

**Pull request merged**